communitygosu.properties

```diff
@@ -1,12 +1,18 @@
 category=Languages
 description=Code Analyzer for Gosu
 homepageUrl=https://github.com/FRI-DAY/sonar-gosu-plugin
-archivedVersions=
-publicVersions=1.2.1
+archivedVersions=1.2.1
+publicVersions=1.2.2
 
 defaults.mavenGroupId=de.friday
 defaults.mavenArtifactId=sonar-communitygosu-plugin
 
+1.2.2.description=Various bug fixes
+1.2.2.sqVersions=[8.9,9.9.*]
```

Comment:
It's time for ranges to begin with the current LTS: 9.9. 
And just to double-check, it's really not compatible with 10.0 and beyond?